src/firefly/js/visualize/ChartsTableViewPanel.jsx

```diff
@@ -527,7 +589,7 @@ export class ChartsTableViewPanel extends Component {
     }
 
     getNextState() {
-        const tblId = TblUtil.getActiveTableId();
+        const tblId = this.props.tblId || TblUtil.getActiveTableId();
```

Comment:
This is not the latest code. Now I have

    getNextState() {
        var {tblId, chartId} = this.props;
        tblId = tblId || chartId ? getTblIdForChartId(chartId) : TblUtil.getActiveTableId();
        chartId = this.props.chartId || tblId;
        const tableModel = TblUtil.getTblById(tblId);

It covers several situations: 
1. tblId is passed as a property: the chart is only connected to a particular table
2. chartId is passed as a property: tblId for a given chart is set in the store (this is the api case when chart is connected to the active table of a given group).
3. both tblId and chartId are empty: current firefly app scenario, chart is connected to the active table of the main group.